eng/common/scripts/Cadl-Project-Generate.ps1

```diff
@@ -14,6 +14,78 @@ $ErrorActionPreference = "Stop"
 . $PSScriptRoot/common.ps1
 Install-ModuleIfNotInstalled "powershell-yaml" "0.4.1" | Import-Module
 
+function GetNpmPackageVersion([string]$packagePath) {
```

Comment:
this should be moved to the new `Typespec-Project-Generate.ps1` as this file will be deleted after all languages swap to the new one.

Reply:
Yes, will move this change to `Typespec-Project-Generate.ps1` after "cadl rename" finished.

Reply:
> Typespec-Project-Generate.ps1

It should be `TypeSpec-Project-Generate.ps1`.